src/main/java/seedu/address/model/person/DefaultPersonComparator.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.model.person;
+
+import static seedu.address.model.person.PersonComparatorUtil.compareAddress;
+import static seedu.address.model.person.PersonComparatorUtil.compareEmail;
+import static seedu.address.model.person.PersonComparatorUtil.compareFavorite;
+import static seedu.address.model.person.PersonComparatorUtil.compareName;
+import static seedu.address.model.person.PersonComparatorUtil.comparePhone;
+
+import java.util.Comparator;
+
+/**
+ * Default comparator for persons. Sorts first by favorites, then by name in alphabetical order,
+ * then by phone in numeric order, then by address in alphabetical order, then by email in alphabetical order
+ */
+public class DefaultPersonComparator implements Comparator<ReadOnlyPerson> {
```

Comment:
As this is a fairly large project, could I suggest the following naming conventions to maintain some sanity, so that despite the list is getting long, we can still find our files easily:

`PersonComparatorDefault`
`PersonComparatorOrderByName`